client/dashboard/app/queries/domains-dnssec.ts

```diff
@@ -0,0 +1,12 @@
+import { mutationOptions } from '@tanstack/react-query';
```

Comment:
[nit] can we update the filename to `domain-dnssec.ts` to indicate that this is about one domain? For consistency with `site-*`.